activerecord/test/cases/persistence_test.rb

```diff
@@ -43,31 +43,42 @@ def test_populates_non_primary_key_autoincremented_column_for_a_cpk_model
     assert_not_nil order_id
   end
 
-  def test_fills_auto_populated_columns_on_creation
-    record_with_defaults = Default.create
-    assert_not_nil record_with_defaults.id
-    assert_equal "Ruby on Rails", record_with_defaults.ruby_on_rails
-
-    if current_adapter?(:PostgreSQLAdapter) && ActiveRecord::Base.connection.supports_virtual_columns?
-      assert_not_nil record_with_defaults.virtual_stored_number
-    end
-
-    assert_not_nil record_with_defaults.random_number
-    assert_not_nil record_with_defaults.modified_date
-    assert_not_nil record_with_defaults.modified_date_function
-    assert_not_nil record_with_defaults.modified_time
-    assert_not_nil record_with_defaults.modified_time_without_precision
-    assert_not_nil record_with_defaults.modified_time_function
+  if current_adapter?(:PostgreSQLAdapter, :SQLite3Adapter, :Mysql2Adapter, :TrilogyAdapter)
```

Comment:
I'm not used to seeing all of these adapters listed -- is this so that we avoid running the tests against `OracleAdapter`, etc?

Reply:
Yes. I have seen recently (can't find that issue/PR) that someone mentioned they use active record tests as part of their adapter tests.